test/utils/assertions.js

```diff
@@ -0,0 +1,11 @@
+var _ = require('lodash');
+var path = require('path');
+var should = require('should');
+should.Assertion.add('equalFileSystemPath', function(value, description){
+    value = value.replace(/\//g, path.sep);
+    if(process.platform == 'win32' && _.startsWith(value, path.sep)){
+        value = __dirname.split(path.sep)[0] + value;
```

Comment:
So as I understand:
1. If value already contain disk name `d:\some\file` - this code will be not executed
2. If value is absolute, but doesn't contain disk name `\some\file` - this code will be executed and it will add current disk name to the start of value

Looks logical. Let's leave it as is if it is needed.